pgmpy/tests/test_models/test_MarkovChain.py

```diff
@@ -107,6 +107,34 @@ def test_add_variable_new(self):
         self.assertEqual(model.cardinalities['p'], 3)
         self.assertDictEqual(model.transition_models['p'], {})
 
+    def test_copy(self):
+        model = MC(['a', 'b'], [2, 2],[State('a', 0), State('b', 1)])
+        model.add_transition_model('a',{0: {0: 0.1, 1: 0.9}, 1: {0: 0.2, 1: 0.8}})
+        model.add_transition_model('b',{0: {0: 0.3, 1: 0.7}, 1: {0: 0.4, 1: 0.6}})
+        copy = model.copy()
+
+        self.assertIsInstance(copy, MC)
+        self.assertIsNot(copy, model)
```

Comment:
@anchitjain1234 Wrong. This test will always pass because `MarkovChain` doesn't have `__eq__` method.